src/Castle.Facilities.EventWiring/Castle.Facilities.EventWiring-VS2017.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net45</TargetFramework>
+   
+    <NoWarn>$(NoWarn);1591</NoWarn>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <AssemblyName>Castle.Facilities.EventWiring</AssemblyName>
+    <RootNamespace>Castle.Facilities.EventWiring</RootNamespace>
+    <AssemblyOriginatorKeyFile>../../buildscripts/CastleKey.snk</AssemblyOriginatorKeyFile> 
+    <SignAssembly>true</SignAssembly> 
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <DefineConstants>$(DefineConstants);DOTNET45</DefineConstants>
```

Comment:
Yes, Alina, this is a work in progress in castle core too. If there is anything you think I should be doing while I am up in core please let me know. It very quickly gets out of control which is why @jonorossi is calling it out.